api/types/event/snapshot.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package containerd.v1.types;
```

Comment:
After looking at this, these should be declared by the services that emit them, rather than the event package. I'm not sure we need a secondary events package. The envelope can be part of the events service and the other events can be part of the services that emit them.